mlflow/server/js/src/experiment-tracking/components/ExperimentPage.js

```diff
@@ -287,6 +314,106 @@ export class ExperimentPage extends Component {
     );
   };
 
+  onClear = () => {
+    // When user clicks "Clear", preserve multicolumn toggle state but reset other persisted state
```

Comment:
Why do we want to maintain the multicolumn toggle state? 
@dbczumar what do you think about this requirement?

Reply:
It's something that was already in place before this PR. Personally I think it makes sense from a UX perspective as well assuming that most users have a preferred view (either the multicolumn view or the regular view). Clicking the clear button then, should not toggle/reset this view

Reply:
I think we should preserve it, given that we did so previously